README.md

```diff
@@ -36,10 +36,11 @@ git clone https://github.com/GoogleCloudPlatform/bank-of-anthos.git
 
 ### 2 - Project setup
 
-[Create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project) or use an existing project. Then, set the Project ID variable.
+[Create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project) or use an existing project. Set the PROJECT_ID environment variable and ensure the Google Kubernetes Engine API is enabled.
 
 ```
 PROJECT_ID=<your-project-id>
+gcloud beta services enable container --project $PROJECT_ID
```

Comment:
nit: we have braces for the container clusters create command
`--project=${PROJECT_ID}`

Reply:
Done!